tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/run_direct_ptx_vision_competitors.py

```
@@ -0,0 +1,376 @@
+                               "method": f"torchvision.ops.{name}",
+                               "reason": "operation is absent from the installed torchvision build"})
+
+    cases.append(Case("BoxArea", "N=4096", "torchvision.ops.box_area",
+                      lambda: ops.box_area(boxes_area), 4096.0, 3.0, 20.0,
+                      lambda: ops.box_area(boxes_area.cpu().double())))
+    cases.append(Case("BoxConvert", "N=4096,xyxy->cxcywh", "torchvision.ops.box_convert",
+                      lambda: ops.box_convert(boxes_area, "xyxy", "cxcywh"),
+                      4096.0, 8.0, 32.0,
+                      lambda: ops.box_convert(boxes_area.cpu().double(), "xyxy", "cxcywh")))
+
+    loss_predicted = boxes(4096, 854).requires_grad_(True)
```

Ruff (0.16.0)

[warning] Boolean positional value in function call

(FBT003)